src/components/transactions/TxDetails/TxData/DecodedData/SingleTxDecoded/index.tsx

```diff
@@ -39,23 +28,19 @@ export const SingleTxDecoded = ({
   expanded,
   onChange,
 }: SingleTxDecodedProps) => {
-  const chain = useCurrentChain()
   const isNativeTransfer = tx.value !== '0' && (!tx.data || isEmptyHexData(tx.data))
   const method = tx.dataDecoded?.method || (isNativeTransfer ? 'native transfer' : 'contract interaction')
-  const amountInWei = tx.value ?? '0'
-
-  let details
-  if (tx.dataDecoded) {
-    details = <MethodDetails data={tx.dataDecoded} addressInfoIndex={txData.addressInfoIndex} />
-  } else if (tx.data) {
```

Comment:
We are now checking `txData.hexData` inside `DecodedData` to render this block. Is this still the same behaviour?